Makefile

```diff
@@ -54,9 +61,30 @@ define make_docker_dev
 		-f docker/Dockerfile.dev ./build
 endef
 
-all: $(SERVICES)
+ADDON_SERVICES = bootstrap cassandra-reader cassandra-writer certs \
+					influxdb-reader influxdb-writer lora-adapter mongodb-reader mongodb-writer \
+					opcua-adapter postgres-reader postgres-writer provision smpp-notifier smtp-notifier \
+					timescale-reader timescale-writer twins
+
+EXTERNAL_SERVICES = vault prometheus
+
+ifneq ($(filter run%,$(firstword $(MAKECMDGOALS))),)
+  temp_args := $(wordlist 2,$(words $(MAKECMDGOALS)),$(MAKECMDGOALS))
+  DOCKER_COMPOSE_DIRECTION := $(if $(filter up down,$(temp_args)), $(filter up down,$(temp_args)), $(DEFAULT_DOCKER_COMPOSE_DIRECTION))
+  $(eval $(DOCKER_COMPOSE_DIRECTION):;@)
+endif
+
+ifneq ($(filter run_addons%,$(firstword $(MAKECMDGOALS))),)
+  temp_args := $(wordlist 2,$(words $(MAKECMDGOALS)),$(MAKECMDGOALS))
+  RUN_ADDON_ARGS :=  $(filter-out up down,$(if $(temp_args),$(temp_args),$(ADDON_SERVICES) $(EXTERNAL_SERVICES)))
+  $(eval $(RUN_ADDON_ARGS):;@)
+endif
+
+FILTERED_SERVICE = $(filter-out $(RUN_ADDON_ARGS), $(SERVICES))
 
-.PHONY: all $(SERVICES) dockers dockers_dev latest release
+all: $(SERVICE)
```

Comment:
```suggestion
all: $(SERVICES)
```

Reply:
Correct command , `args` is a make variable 
```bash 
make run args="-d"
```

Reply:
Done changed to `SERVICES`